plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1417,7 +1467,14 @@ function( $subscription ) use ( $site_id ) {
 		);
 		
 		foreach ( $subscriptions as &$subscription ) {
-			$subscription['active'] = in_array( $site_id, $subscription['connections'] );
+			$local   = wp_list_filter(
+				array_merge( $local_plugins, $local_themes ),
+				array( 'slug' => $subscription['zip_slug'] )
+			);
+			$local = array_shift( $local );
+			$org_product = ! empty( $local ) && isset( $local['woo_owned'] ) && $local['woo_owned'] === false;
+
+			$subscription['active'] = in_array( $site_id, $subscription['connections'] ) || $org_product;
```

Comment:
Should we change the active status here or should we set the product as connected on WCCOM?

Reply:
Ya, I would suggest WCCOM as the source of truth.

Reply:
I removed this part and will add it to the WCCOM API later.